src/pykeen/stoppers/early_stopping.py

```diff
@@ -76,6 +76,8 @@ class EarlyStopper(Stopper):
     evaluator: Evaluator
     #: The triples to use for evaluation
     evaluation_triples_factory: Optional[TriplesFactory]
+    #: Current epoch
+    current_epoch: Optional[int] = None
```

Comment:
when does this get updated? shoudn't this be a property?

Reply:
I forgot to push the last changes. Please check 6133b880141b286e98c6409f354add33fd88e00a and c9b78bf92841e7eb82e97093d2e466d68a1185f0.